tools/eslint-changed.js

```diff
@@ -151,9 +151,12 @@ async function main() {
 
 		args = [ 'diff' ];
 		if ( argv.gitBase !== undefined ) {
-			origRef = argv.gitBase;
+			const args2 = [ 'merge-base', argv.gitBase, 'HEAD' ];
+			debug( 'Running git merge-base command:', git, args2.join( ' ' ) );
```

Comment:
Confirming that you'd like the debug output to remain here?

Reply:
Yes, it's intentional. See elsewhere in the file where this same pattern is followed for other git commands.